components/coreapi/utxo.go

```diff
@@ -28,25 +29,56 @@ func getOutputMetadata(c echo.Context) (*outputMetadataResponse, error) {
 		return nil, err
 	}
 	outputID := output.OutputID()
-	slotCommitment := deps.Protocol.MainEngineInstance().Storage.Settings().LatestCommitment()
-	spent, err := deps.Protocol.MainEngineInstance().Ledger.IsOutputSpent(outputID)
+	latestCommitment := deps.Protocol.MainEngineInstance().Storage.Settings().LatestCommitment()
+	includedCommitment, err := deps.Protocol.MainEngineInstance().Storage.Permanent.Commitments().Load(output.BlockID().Index())
```

Comment:
You should use `output.SlotIndexBooked()` here for now. But note, this might fail if the output is in the mempool but hasn't been committed yet.